virtualization_15.m

```diff
@@ -0,0 +1,19 @@
+//
+//  virtualization_15.m
+//
+#import "virtualization_15.h"
+
+/*!
+ @abstract Set nestedVirtualizationEnabled. The default is false.
+ */
+void setNestedVirtualizationEnabled(void *config, bool nestedVirtualizationEnabled)
+{
+#ifdef INCLUDE_TARGET_OSX_15
+    if (@available(macOS 15, *)) {
+        VZGenericPlatformConfiguration *platformConfig = (__bridge VZGenericPlatformConfiguration *)config;
```

Comment:
Not sure `__bridge` is required here? I don't see it used anywhere else in this repo. Apart from this comment, looks good to me!

Reply:
This has been addressed.